libraries/helpers_sensubackend.rb

```diff
@@ -0,0 +1,28 @@
+module SensuCookbook
+  module Helpers
+    module SensuBackend
+      def sensu_backend_bin
+        if node['platform'] != 'windows'
+          '/usr/sbin/sensu-backend'
+        else
```

Comment:
Backend is only available on windows, this has no function.

Reply:
you mean available only on linux?

Reply:
`sensu_backend_bin` does not exist on windows as the platform is not supported for the backend. There's no reason to have the if else here.

Reply:
Thanks, I've removed this section.